pkg/kubelet/cm/dra/cdi.go

```diff
@@ -0,0 +1,272 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dra
```

Comment:
can we note here that this is lifted from https://github.com/container-orchestrated-devices/container-device-interface/blob/main/pkg/cdi/annotations.go and https://github.com/container-orchestrated-devices/container-device-interface/blob/main/pkg/cdi/qualified-device.go ?

Reply:
and https://github.com/container-orchestrated-devices/container-device-interface/blob/main/pkg/cdi/annotations.go as well looks like?

Reply:
Done. I also created and linked to an issue: https://github.com/container-orchestrated-devices/container-device-interface/issues/97

Reply:
@pohly was this part of your effort to remove the vendored dependencies yesterday? I don't recall reviewing this file in my reviews last week.

Reply:
Yes.

Reply:
Thanks for adding note.